gl/float.c

```diff
@@ -20,6 +20,12 @@
 /* Specification.  */
 #include <float.h>
 
+union gl_long_double_union
```

Comment:
needed only for PowerPC support? add under #if then?

Reply:
Yes, it is only needed for Power PC. I will wrap the changes under #if.